src/mumble/AudioOutputUser.h

```diff
@@ -26,9 +27,10 @@ class AudioOutputUser : public QObject {
 	AudioOutputUser(const QString &name);
 	~AudioOutputUser() Q_DECL_OVERRIDE;
 	const QString qsName;
-	float *pfBuffer = nullptr;
-	float *pfVolume = nullptr;
-	float fPos[3]   = { 0.0, 0.0, 0.0 };
+	float *pfBuffer        = nullptr;
+	float *pfVolume        = nullptr;
```

Comment:
Running clang-format on this source file again will probably reduce the indentation of the equal signs to their previous levels again. I assume this is what currently makes the CI fail.